public/pages/ConfigureModel/components/CategoryField/CategoryField.tsx

```diff
@@ -155,7 +156,7 @@ export function CategoryField(props: CategoryFieldProps) {
                     onChange={(options) => {
                       const selection = options.map((option) => option.label);
                       if (!isEmpty(selection)) {
-                        if (selection.length <= 2) {
+                        if (selection.length <= 5) {
```

Comment:
would it be possible that we base this number on our plugin setting and we change the default to 5 there as max?
https://github.com/opensearch-project/anomaly-detection/blob/main/src/main/java/org/opensearch/ad/settings/ADNumericSetting.java#L40